### ISSUE TITLE
where to load the generated pseudo labels of target data?

### ISSUE THREAD

**Issue by jingzhengli** [microsoft/ProDA#14]
First I run the generate_presudo_label.py, then when I run the train.py. I don't know how to load the generated pseudo labels of target data to became **target_train_loader**? could you please show me where this part of the code is?

**Comment by super233** [comment]
Maybe set the parameter `--path_soft` can fix it, you can read `parser_train.py` for details.

https://github.com/microsoft/ProDA/blob/9ba80c7dbbd23ba1a126e3f4003a72f27d121a1f/parser_train.py#L36
https://github.com/microsoft/ProDA/blob/9ba80c7dbbd23ba1a126e3f4003a72f27d121a1f/parser_train.py#L80

**Comment by jingzhengli** [comment]
> Maybe set the parameter `--path_soft` can fix it, you can read `parser_train.py` for details.
> 
> https://github.com/microsoft/ProDA/blob/9ba80c7dbbd23ba1a126e3f4003a72f27d121a1f/parser_train.py#L36
> 
> 
> https://github.com/microsoft/ProDA/blob/9ba80c7dbbd23ba1a126e3f4003a72f27d121a1f/parser_train.py#L80

Thanks for your reply. I just found it in **def __getitem__(self, index):**
